### ISSUE TITLE
enable MacOS build

### ISSUE THREAD

**Issue by notestaff** [conda-forge/git-annex-feedstock#7]
Once travis-ci conda-forge builds for MacOS are working again, we should try to make the git-annex recipe work on MacOS.  See https://github.com/conda-forge/staged-recipes/pull/4605 for earlier attempts.

**Comment by yarikoptic** [comment]
No, AFAIK the situation is quite stable

**Comment by notestaff** [comment]
stable as in, no advance :) worse is that the recent conda-forge changes broke the linux recipe: https://github.com/conda-forge/git-annex-feedstock/pull/32#issuecomment-459119177 .  I haven’t been able to fix it despite many hours; would be grateful for any help.

**Comment by yarikoptic** [comment]
May be the (suboptimal) way forward would be to just redistribute binary build (like `nodep` for linux already doing already) for OSX for now at least?

**Comment by tdegeus** [comment]
Hi! Any updates on this? Any hope that osx (incl. arm) could be supported?

**Comment by yarikoptic** [comment]
Hope is always there but no time/progress to see it through. Indeed not sure what it would take ATM. Last attempt of a kind was in https://github.com/conda-forge/git-annex-feedstock/pull/107 and may be #154 would be of help if finished

**Comment by tdegeus** [comment]
Thanks, let's see. Can you also rerender ?

**Comment by scratchings** [comment]
Is there any news on whether the macOS build will be instated?

**Comment by yarikoptic** [comment]
Unfortunately I personally would not have a chance/motivation for embarking on that ATM.  But note that there are now pypi distributed whl's of git-annex: https://pypi.org/project/git-annex/#files  -- tried those? if you see how to address here, PR would be welcome.

**Comment by scratchings** [comment]
Yes, I'm using that mechanism, but it would be neater if it was in Conda-forge as then installation could all be handled in pixi rather than a mixture of pixi and uv (pixi complains about a platform miss-match which uv ignores).
